browser-extension/README.md

```diff
@@ -70,9 +69,25 @@ Given a version number `MAJOR.MINOR.PATCH`, increment the:
 To release the extension on Chrome or Mozilla Firefox, run the script `./build.sh`
 and upload the generated .zip file in the corresponding extension web store.
 
-## License
+## Copyright & License
+
+Copyright 2021-2022 Association Tournesol and contributors.
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-See the [LICENSE.md](./src/LICENSE) file.
+Included license:
+ - [AGPL-3.0-or-later](./LICENSE)
```

Comment:
`src` is the folder packed by our build command right?

I don't know how the Chrome and Firefox extensions stores work, but it could be good to have the license included if the stores try to guess the license by parsing the sources.

I'll do a bit of research.